tests/unit/metadata/test_multi_table.py

```diff
@@ -385,8 +385,11 @@ def test__validate_child_map_circular_relationship(self):
         }
 
         # Run / Assert
-        # TODO fix match error message
-        with pytest.raises(ValueError):
+        err_msg = (
+            'The relationships in the dataset describe a circular dependency between '
+            "tables {'users', 'sessions'}."
```

Comment:
Did we ever decide how to deal with these?

Reply:
@pvk-developer was going to write some functionality for this, but honestly I think it would be easier to just do 
```python
f'Example string with set {sorted(set_variable)}'
```
everywhere with sets, granted this would print it with square brackets